.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         # Docker/Moby still builds runc with Go 1.13, so we should still support Go 1.13.
```

Comment:
Now we can drop 1.13

Reply:
I know Moby moved on, but I'm not sure about everybody else.

This is a maintenance branch, perhaps it makes sense to keep 1.13 compatibility since the cost is zero (unless we'll want to backport something massive) and it might save someone a big headache.

Reply:
Yeah, the comment is no longer "correct", but it I guess it's fine to keep the `1.13` version in the test-matrix for now (it's just a single word).